impeller/renderer/formats.h

```diff
@@ -86,8 +87,6 @@ enum class BlendOperation {
   kAdd,
   kSubtract,
   kReverseSubtract,
-  kMin,
-  kMax,
```

Comment:
This reminds me: At some point we gotta talk about a strategy for extensions. We can make all the advanced blends done in-pipeline with [EXT_blend_minmax](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_blend_minmax.txt) + [KHR_blend_equation_advanced](https://www.khronos.org/registry/OpenGL/extensions/KHR/KHR_blend_equation_advanced.txt) for GLES and avoid the extra render passes down the road.

Reply:
Thats a good point. From reading the spec, I believe `KHR_blend_equation_advanced_coherent` will be necessary right? In any case, I left these out because I could not find anything similar for Metal or Vulkan (and hadn't considered other API's like DX12). If we are going to need to work with these extensions anyway, I'd rather leave them out now and devise a separate strategy for extensions. I really don't want to be super prescriptive about not using extensions though. If they are available and avoid render passes, then I'm all for it. Could you please file a bug for this?

Reply:
SGTM, this is mentioned in https://github.com/flutter/flutter/issues/100086 but probably belongs in a separate issue.